test/misc.jl

```diff
@@ -55,3 +55,16 @@ end
         end
     end
 end
+
+@testset "binomlogpdf" begin
+    @testset "identity" begin
+        for p in [rand(50); 0; 1]
+            for n in 0:50
+                for k in [-50:50; randn(10)]
+                    # testing genetic Julia binompdf against Rmath version.
+                    @test Float32(binomlogpdf(n, Float32(p), k)) ≈ Float32(log(binompdf(n, p, k)))
```

Comment:
```suggestion
                    @test Float32(binomlogpdf(n, Float32(p), k)) ≈ Float32(logbinompdf(n, p, k))
```
will still use the Rmath version and be less prone to overflow so you can test your version for larger `n` and `k`.